tools/cert-sidecar/config/default.conf

```diff
@@ -0,0 +1,40 @@
+
+#URL of the x509 identity mgmt and Resources
+x509.url:string=http://x509-identity-mgmt:3000
+x509.path.sign:string=/internal/api/v1/throw-away
+x509.path.crl:string=/internal/api/v1/throw-away/ca/crl
+x509.path.ca:string=/internal/api/v1/throw-away/ca
+x509.retries:integer=9
+x509.timeout:integer=1000
+x509.healthchecker:integer=60000
+
+#Geral config
+app.sidecar.to:string=app
+
+#Logger config
+log.console.level:string=info
+log.console.verbose:boolean=false
+log.file:boolean=false
+log.file.level:string=debug
+log.file.filename:string=cert-sidecar-logs-%DATE%.log
```

Comment:
Forgot to mention that we are trying to use the same names for all the services. In Kafka WS and V2K we are using different names for the logger, example:
logger.transports.console.level
I think it will be nice for you to use the Kafka WS configuration names.

Reply:
Ok, I will change, but I think `logger.transports.console.level` is so long...

Reply:
After some discussions, we decided to keep @mprevide patterns for the logger because of its brevity.